ios/App/Shared/player/AudioPlayer.swift

```diff
@@ -104,9 +104,6 @@ class AudioPlayer: NSObject {
     }
     
     deinit {
-        self.stopPausedTimer()
-        self.removeSleepTimer()
-        self.removeTimeObserver()
```

Comment:
These needed to be moved to `destroy()`, as when setting `player = nil`, `removeSleepTimer` will throw an event that causes `PlayerHandler.player` to be read, which is causes an access crash due to a mutated variable being read.